lib/rubocop/ast/processed_source.rb

```diff
@@ -229,9 +229,9 @@ def parser_class(ruby_version)
         when 2.7
           require 'parser/ruby27'
           Parser::Ruby27
-        when 2.8
-          require 'parser/ruby28'
-          Parser::Ruby28
+        when 3.0
```

Comment:
could we have `when 2.8, 3.0` for compatibility?

Reply:
Ah, I get it. In the future, the syntax of Ruby 2.8 and Ruby 3.0 may be dissociated, but I think they are almost the same now. It seems better that both are still supported as migration path. I updated this one.

Reply:
Well, this way we are compatible with older rubocop who know about 2.8 and not about 3.0. There won't be an actual Ruby 2.8 afaik.